apps/parsley/src/gql/fragments/base-task.graphql

```diff
@@ -3,7 +3,7 @@ fragment BaseTask on Task {
   execution
   id
   patchNumber
-  status
+  status: displayStatus
```

Comment:
for example here, it seems like we're not using the `displayStatus` field at all even though it was introduced in the backend PR